release-doc-in-process/all_vf.TTL

```diff
@@ -198,13 +198,13 @@ vf:relationship  a          owl:ObjectProperty ;
         vs:term_status      "unstable" ;
         rdfs:comment        "A kind of relationship that exists between 2 agents." .
 
-vf:affects
-        a                   owl:ObjectProperty ;
-        rdfs:label          "affects" ;
-        rdfs:domain         vf:EconomicEvent ;
-        rdfs:range          vf:EconomicResource ;
-        vs:term_status      "unstable" ;
-        rdfs:comment        "The economic resource that is decremented or incremented by the economic event, or just what the economic event is about." .
+#vf:affects
+#        a                   owl:ObjectProperty ;
+#        rdfs:label          "affects" ;
+#        rdfs:domain         vf:EconomicEvent ;
+#        rdfs:range          vf:EconomicResource ;
+#        vs:term_status      "unstable" ;
+#        rdfs:comment        "The economic resource that is decremented or incremented by the economic event, or just what the economic event is about." .
 
```

Comment:
Let's just delete it, we can always find it in git history so we have 'undo' button there if needed.

Reply:
done